chcemvediet/apps/accounts/forms.py

```diff
@@ -65,15 +69,70 @@ def signup(self, request, user):
 class SettingsForm(forms.Form):
 
     anonymize_inforequests = forms.BooleanField(
-        label=_(u'accounts:SettingsForm:anonymize_inforequests:label'),
-        required=False,
-    )
+            label=_(u'accounts:SettingsForm:anonymize_inforequests:label'),
+            required=False,
+            )
+
+    custom_anonymization = forms.BooleanField(
+            label=_(u'accounts:SettingsForm:custom_anonymization:label'),
+            required=False,
+            widget=forms.CheckboxInput(attrs={
+                u'class': u'pln-toggle-changed',
+                u'data-container': u'form',
+                u'data-hide-target-true': u'.form-group:has(.chv-visible-if-custom-anonymization)',
+                u'data-disable-target-true': u'.chv-visible-if-custom-anonymization',
+            }),
+            )
+
+    help_text = _(u'accounts:SettingsForm:custom_anonymized_strings:help_text')
+    plural_form = plural(WORD_SIZE_MIN, u'1:znak', u'2~4:znaky', u'znakov')
```

Comment:
@viliambalaz Takto lokalizacie mnoznych cisel robit nemozeme. Lebo takyto `plural_form` nemas ako lokalizovat, kedze slovenske znenie je hardcodovane v zdrojaku.

Ale popravde neviem, ako sa to spravne robi v Djangu. Skus pogooglit. Malo by fungovat `ungettext` s:
```
msgid "accounts:SettingsForm:custom_anonymized_strings:help_text."
msgid_plural "accounts:SettingsForm:custom_anonymized_strings:help_text %(count)s."
msgstr[0] "Slovné spojenie, ktoré sa anonymizuje, musí mať minimálne %(count)s znakov."
msgstr[1] "Slovné spojenie, ktoré sa anonymizuje, musí mať minimálne %(count)s znak."
msgstr[2] "Slovné spojenie, ktoré sa anonymizuje, musí mať minimálne %(count)s znaky."
...
```
Ale neskusal som a nasiel som aj nejake diskusie, ze to nefunguje.

Nejak hadam Django ma vymyslene ako spravne lokalizovat mnozstne cisla cez gettext. Inac by to bolo useless na lokalizaciu.